{{cookiecutter.project_name}}/databricks-config/README.md

```diff
@@ -92,12 +92,12 @@ The model validation stack focuses on building a plug-and-play stack component f
 in staging and prod.
 Its central purpose is to evaluate a registered model and validate its quality before deploying the model to Production/Staging.
 
-The model validation job is implemented in `notebooks/ModelValidation`. The model validation stack is defined in 
-`staging/training-job.tf` and `prod/training-job.tf`.
+The model validation job is implemented in `notebooks/ModelValidation](../notebooks/ModelValidation.py). The model validation stack is defined in 
```

Comment:
```suggestion
The model validation job is implemented in notebooks/ModelValidation](../notebooks/ModelValidation.py). The model validation stack is defined in 
```

Reply:
WOW great catch! It wasn't a valid link format and did not get  highlighted by intelliJ.

Reply:
Haha glad my annoying nit-picking tendencies came in handy this time